localization/src/utils/getters.ts

```diff
@@ -52,9 +54,17 @@ export function getDateFormatInfo(): DateFormatInfo {
     localePattern = `${localePattern}${symbols[components[1]]}`;
     localePattern = `${localePattern}${separator}${symbols[components[2]]}`;
 
+    // Determining the first day of the week by locale when using library can consume too much space,
+    // but without it it will be inaccurate,
+    // so for simplicity we will define that on the web all weeks start on Monday, as the most common day.
+    // @ts-expect-error
+    const dayOfWeek = dayjs().weekday(0);
```

Comment:
Why it has an error? What about the default value (for non-web platforms)?

Reply:
@sertony okay, this PR still in work :)

Reply:
Ok, I've marked it as draft then ;)